ghost/core/core/server/services/mega/template.js

```diff
@@ -1,15 +1,43 @@
 /* eslint indent: warn, no-irregular-whitespace: warn */
 const iff = (cond, yes, no) => (cond ? yes : no);
+const sanitizeHtml = require('sanitize-html');
+
+/**
+ * @template {Object.<string, any>} Input
+ * @param {Input} obj
+ * @param {string[]} [keys]
+ * @returns {Input}
+ */
+const sanitizeKeys = (obj, keys) => {
+    const sanitized = Object.assign({}, obj);
+    const keysToSanitize = keys || Object.keys(obj);
+
+    for (const key of keysToSanitize) {
+        if (typeof sanitized[key] === 'string') {
+            // @ts-ignore
+            sanitized[key] = sanitizeHtml(sanitized[key], {
+                allowedTags: false,
+                allowedAttributes: false
+            });
+        }
+    }
+
+    return sanitized;
+};
+
 module.exports = ({post, site, newsletter, templateSettings}) => {
     const date = new Date();
     const hasFeatureImageCaption = templateSettings.showFeatureImage && post.feature_image && post.feature_image_caption;
+    const cleanPost = sanitizeKeys(post, ['title', 'excerpt', 'html']);
```

Comment:
I've found we do need to sanitize the `html` property in case in includes something like `'<'`.

I think this should be fine as we don't strip any tags or attributes